x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -132,13 +289,13 @@ export const registerCollector: RegisterCollector = ({
         },
       },
     },
-    isReady: () => kibanaIndex.length > 0,
+    isReady: () => true,
```

Comment:
just curious why this check is no longer needed?

Reply:
See this comment: https://github.com/elastic/kibana/pull/95659#discussion_r614463838